src/mono/wasm/build/WasmApp.InTree.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)WasmApp.props" />
+  <PropertyGroup>
+    <Platform>AnyCPU</Platform>
+    <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
+    <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
+    <MicrosoftNetCoreAppRuntimePackRidDir>$(ArtifactsBinDir)microsoft.netcore.app.runtime.browser-wasm\$(Configuration)\runtimes\browser-wasm\</MicrosoftNetCoreAppRuntimePackRidDir>
+    <EMSDK_PATH Condition="'$(EMSDK_PATH)' == '' and '$(MonoProjectRoot)' != ''">$(MonoProjectRoot)wasm\emsdk</EMSDK_PATH>
```

Comment:
Would we have to worry about `'$(EMSDK_PATH)' == ''` and `'$(MonoProjectRoot)' == ''`? Should there be a warning message for this case?

Reply:
`InTree.*` are meant to be used within the tree, where you have the root `Directory.Build.*` being imported too, so this shouldn't be a problem. But I'll add a comment on the files about this.